test/functional/autoupdate/aup-basic.bats

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	if [ -z "$TRAVIS" ]; then
+		skip "This test is intended to run only in Travis (use TRAVIS=true to run it anyway)..."
+	fi
+
+	create_test_environment "$TEST_NAME"
+
+}
+
+test_teardown() {
+
+	if [ -z "$TRAVIS" ]; then
+		return
+	fi
+
+}
+
+@test "AUT001: Basic test, Check auto-update enable" {
+
+	# perform autoupdate disable
+	run sudo sh -c "$SWUPD autoupdate --disable --path=$PATH_PREFIX"
```

Comment:
where are you defining PATH_PREFIX?

Reply:
This is created as part of the test environment ..testlib

Reply:
right I don't know how I missed that, I see it know, thanks